.github/workflows/scip-release.yml

```diff
@@ -31,4 +31,4 @@ jobs:
         with:
           file: Dockerfile.autoindex
           push: true
-          tags: sourcegraph/scip-python:autoindex
+          tags: sourcegraph/scip-python:autoindex,sourcegraph/scip-python:${{ github.ref_name }}
```

Comment:
The autoindex tag is for marking the image for [auto-indexing](https://docs.sourcegraph.com/code_intelligence/explanations/auto_indexing). In this particular repo, there is only one Dockerfile so there isn't a difference, but some other indexers have two different Dockerfiles.

For instance, the [scip-typescript auto-indexing image](https://github.com/sourcegraph/scip-typescript/blob/main/Dockerfile.autoindex#L6) includes a C++ compiler to be able to successfully `npm install` more packages and make more progress for indexing, which isn't part of the `latest` image.

Reply:
For now, I think it is OK to add a `latest` tag here. We can later revisit if we'd like to split the auto-indexing vs non-auto-indexing images. A bunch of numerics-related Python packages do rely on C/C++/Fortran libraries, so it might make sense to bundle in compilers for those in the auto-indexing image in the future.